src/app/api/notify/route.ts

```diff
@@ -28,6 +28,11 @@ export async function POST(request: NextRequest, response: NextResponse) {
             body: JSON.stringify(notificationPayload),
         })
     } catch (error: any) {
+        console.error('Error occured while sending notification:', error)
         throw new Error('Error occured while sending notification')
     }
+
+    return NextResponse.json({
+        status: 200,
+    })
```

Comment:
Why is this necessary?

Reply:
I think it's how nextjs works, not related to `fetch`. Function must return something successful, otherwise next thinks something went wrong and returns a 500 to the user

Reply:
Nextjs wants all api calls to happen from the api folder. This is handles as an actual 'next api', so also expects an next api response. Thanks for introducing this alexey